src/main/java/Duke.java

```diff
@@ -1,10 +1,140 @@
-public class Duke {
-    public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
-    }
-}
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Scanner;
+
+public class Duke {
+    
+    private static void save(ArrayList<String> tasks) throws IOException {
+        File f = new File("tasks.txt");
+        System.out.print(f.createNewFile());
+        FileWriter fw = new FileWriter("tasks.txt");

```

Comment:
A longer, more descriptive name (like simply `fileWriter`) might be better here.